src/common/com/intellij/plugins/haxe/compilation/HaxeCompilerServices.java

```diff
@@ -290,8 +290,7 @@ private int recalculateFileOffset(@NotNull PsiFile file, @NotNull PsiElement pos
         timeLog.stamp("Compiler finished. Output found on " + (stdout.isEmpty() ? "" : "stdout ") + (stderr.isEmpty() ? "" : "stderr"));
         // LOG.debug("Compiler finished. Output found on " + (stdout.isEmpty() ? "" : "stdout ") + (stderr.isEmpty() ? "" : "stderr"));
         if (0 != status) {
```

Comment:
Changes in this file have to do with parsing the completion output even when an error is returned from the compiler.  The completion output generally is provided before any real error messages.  When that happens, we pull the completion results out of the stream and then send those on to the result parser.  We ignore error output in that case.